package.json

```diff
@@ -63,11 +63,10 @@
     "rimraf": "^2.6.2",
     "single-spa": "^5.1.1",
     "ts-jest": "^24.0.0",
-    "typescript": "3.2.4",
+    "typescript": "3.7.5",
     "webpack": "^4.29.6"
   },
   "dependencies": {
-    "semver": "^6.2.0",
-    "single-spa": ">= 5.1.1"
```

Comment:
Why is single-spa removed as a dependency?

Reply:
It is in two places - dependencies and devDependencies. I accidentally removed it from dependencies, but actually have to remove from devDepedencies. I will revert.